Constitution.txt

```diff
@@ -1,490 +1,264 @@
-  Rufus King
-
-
-Connecticut
-  Wm.   Saml. Johnson
-  Roger Sherman
-
-
-New York
-  Alexander Hamilton
-
-
-New Jersey
-  Wil. Livingston
-  David Brearley.
-  Wm. Paterson. 
-  Jona: Dayton
-
-
-Pennsylvania
-  B Franklin
-  Thomas Mifflin
-  Robt Morris
-  Geo. Clymer
-  Thos. FitzSimons 
-  Jared Ingersoll 
-  James Wilson. 
-  Gouv Morris
-
-
-
-Letter of Transmittal
-skip to Letter of Transmittal to Congress   up to the Constitution
-In Convention. Monday September 17th 1787.
-Present
-The States of
-New Hampshire, Massachusetts, Connecticut, Mr. Hamilton from New York, New Jersey, Pennsylvania, Delaware, Maryland, Virginia, North Carolina, South Carolina and Georgia.
+1: The government of any State may, at its discretion and with the assent of its Citizens, according to the State's constitution, elect to leave the Union, and inform the President of these United States in writing, signed by the State's chief executive. 
 
-Resolved, That the preceeding Constitution be laid before the United States in Congress assembled, and that it is the Opinion of this Convention, that it should afterwards be submitted to a Convention of Delegates, chosen in each State by the People thereof, under the Recommendation of its Legislature, for their Assent and Ratification; and that each Convention assenting to, and ratifying the Same, should give Notice thereof to the United States in Congress assembled. Resolved, That it is the Opinion of this Convention, that as soon as the Conventions of nine States shall have ratified this Constitution, the United States in Congress assembled should fix a Day on which Electors should be appointed by the States which shall have ratified the same, and a Day on which the Electors should assemble to vote for the President, and the Time and Place for commencing Proceedings under this Constitution. 
-That after such Publication the Electors should be appointed, and the Senators and Representatives elected: That the Electors should meet on the Day fixed for the Election of the President, and should transmit their Votes certified, signed, sealed and directed, as the Constitution requires, to the Secretary of the United States in Congress assembled, that the Senators and Representatives should convene at the Time and Place assigned; that the Senators should appoint a President of the Senate, for the sole Purpose of receiving, opening and counting the Votes for President; and, that after he shall be chosen, the Congress, together with the President, should, without Delay, proceed to execute this Constitution.
+2: If the exiting State has land borders with a territory which is not a member of the Union, or has water access to the Oceans of the World through waters not claimed as territory by any other State of the Union, the State shall be considered to be sovereign and foreign from these United States on the date one year after the President of these United States has been informed of the State's intent to leave the Union.
```

Comment:
Brexit has seemed an interminable process, in no small part because both the Parliament of the UK and the Council of the EU have been reluctant to see the process through, instead voting to extend the deadline multiple times.

Since the process for deciding to leave will have been clear, the decision will have been made final when the notice of exit is delivered to the President. Therefore, there must be a hard deadline of transition, with significant leadup time.